pkg/v0/cmd.go

```diff
@@ -26,6 +27,11 @@ const (
 	githubRepo = "operator-framework-olm"
 )
 
+var depRepos = []string{
+	"operator-framework/api",
+	"operator-framework/operator-registry",
+}
+
```

Comment:
Nit: define this somewhere non-global. Maybe in the Options struct? Or maybe in the `Run` command, and then pass through function signatures.

Reply:
This is used in exactly one place in the code, and could simply be moved there? Although I like this _type_ of stuff at the beginning so it can be more easily modified (which we should never have to do _here_).